### PR TITLE
404 Page Design 

### PR DESCRIPTION
**Pull request by yashfalke77**
Hey @EmmaDawsonDev , I Completed the 404 page design below are the screenshots. If any changes are required do tag me :)
image

<a href="https://gitpod.io/#https://github.com/AccessibleForAll/AccessibleWebDev/pull/66"></a>

### CODE REVIEW

File: pages/404.tsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
+/* eslint-disable @next/next/no-img-element */
+import React from "react"
+import styles from "../styles/pageNotFound.module.css"
+import notFoundImage from "../public/404.png"
+import Link from "next/link"
+
+const PageNotFound = () => {
+	return (
+		<div className={styles.notFound}>
```
Comment (EmmaDawsonDev) [inline pages/404.tsx:9]:
```suggestion
		<main className={styles.notFound}>
```

File: styles/pageNotFound.module.css [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+    display: flex;
+    justify-content: flex-end;
+}
+
+.linkItems {
+    padding: 0.4rem;
+}
+
+.link {
+    text-decoration: none;
```
Comment (EmmaDawsonDev) [inline styles/pageNotFound.module.css:50]: Please add the text decoration back in. I would prefer them to be underlined to signal that they are links.

File: styles/pageNotFound.module.css [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+    transition: all 0.3s ease-in-out;
+}
+
+.link:hover {
+    color: #00b4d8;
+}
+
+@media only screen and (max-width: 37.5em) {
```
Comment (EmmaDawsonDev) [inline styles/pageNotFound.module.css:58]: The image is looking a bit too small on ipad screen sizes. Can we change this media query so it goes to 1 column even on ipad?

**Review by EmmaDawsonDev — CHANGES_REQUESTED** [review]
I really like how this is looking. There are just a few small changes to make before merging. Let me know if you need help with anything.

Please remove package-lock and yarn-lock files from the commits (you will need to use git reset or git revert for this)

**Comment by yashfalke77** [comment]
hey @EmmaDawsonDev done all the fixes except of the removal of package lock and yarn lock , I didn't get it how to do , Can you give the procedure how to do it ?

**Comment by EmmaDawsonDev** [comment]
> hey @EmmaDawsonDev done all the fixes except of the removal of package lock and yarn lock , I didn't get it how to do , Can you give the procedure how to do it ?

You need to revert the commit where they were added by typing ``` git revert 8ddf171``` in the terminal. This will ask you to make a revert commit which you can go ahead and save. Then you can re-add only the necessary files by using ```git add filename1 filename2 filename3``` and commit those as normal with ```git commit -m "Message"``` then git push and the changes should appear here.

**Comment by yashfalke77** [comment]
@EmmaDawsonDev commit id would be the same 8ddf171 or I should check the commit id, I am little bit new to git and gothub don't have that much knowledge :)

**Comment by EmmaDawsonDev** [comment]
That should be the correct git id

**Comment by yashfalke77** [comment]
Hey @EmmaDawsonDev made all the changes according to your recommendations, do checkout ..

**Review by EmmaDawsonDev — CHANGES_REQUESTED** [review]
Getting closer :) the package-lock file is now gone but I think you may have deleted the yarn.lock file completely instead of just removing it from your commit. (Don’t worry, it happens often that people confuse the words remove and delete and files disappear). you can copy the contents of the yarn.lock file from this repo and paste back into your branch then commit again.

**Comment by yashfalke77** [comment]
@EmmaDawsonDev added yarn lock file

**Comment by EmmaDawsonDev** [comment]
Unfortunately it's still not the correct yarn.lock file. I think instead of messing around with git revert it would be best if you create a new branch, copy the changes over to the three files needed and only add and commit those files (never use ```git add . ``` in the terminal as you get files like the lock files added when they shouldn't be there. From there you can make a new pull request with that branch to this repo.

**Comment by EmmaDawsonDev** [comment]
> Unfortunately it's still not the correct yarn.lock file. I think instead of messing around with git revert it would be best if you create a new branch, copy the changes over to the three files needed and only add and commit those files (never use `git add . ` in the terminal as you get files like the lock files added when they shouldn't be there. From there you can make a new pull request with that branch to this repo.

Actually, ignore me I managed to fix it.

**Pull request merged**